docs/content/en/docs/concepts/tracing-policy/hooks.md

```diff
@@ -156,6 +156,104 @@ spec:
     - index: 4
       type: "int64"
 ```
+## Raw Tracepoints
+
+Raw tracepoints allow same attachment as tracepoints, but allow access to raw
+tracepoint arguments (compared to standard tracepoints that provide access to
+arguments through tracepoint format).
```

Comment:
nit: maybe we can explain why raw_tracepoint are interested compared to tracepoint (performance)?